bin/hhBtagTest.cc

```diff
@@ -1,12 +1,13 @@
 #include <map>
 #include "../interface/HH_BTag.h"
 
-int main()
+int main(int argc, char *argv[])
 {
+    std:string version = "v1" // model version "v1" or "v2"
```

Comment:
- take the model version from the command line so that one can test both
- as it is now, the code doesn't compile